pkgs/tools/misc/tmux/default.nix

```diff
@@ -1,10 +1,11 @@
 { stdenv, fetchurl, ncurses, libevent, pkgconfig }:
 
 stdenv.mkDerivation rec {
-  name = "tmux-2.0";
+  name = "tmux-${version}";
+  version = "2.0";
 
   src = fetchurl {
```

Comment:
I don't think it applies to `/releases/` urls, they can be any tarballs that the maintainers upload, i.e. different from the tarballs generated from git. (Maybe that's a fetchzip bug?)

Changing to fetchzip actually breaks the build, presumably because the timestamp of `configure` is now older than `configure.ac`:
````
config.status: executing depfiles commands
building
make flags: SHELL=/nix/store/hqirnxfl9cnw0myjlkaasn93q235i18q-bash-4.3-p33/bin/bash    
CDPATH="${ZSH_VERSION+.}:" && cd . && /nix/store/hqirnxfl9cnw0myjlkaasn93q235i18q-bash-4.3-p33/bin/bash /tmp/nix-build-tmux-2.0.drv-0/tmux-2.0.tar.gz/etc/missing aclocal-1.14 
/tmp/nix-build-tmux-2.0.drv-0/tmux-2.0.tar.gz/etc/missing: line 81: aclocal-1.14: command not found
WARNING: 'aclocal-1.14' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
````